winconf.pri

```diff
@@ -1,18 +1,16 @@
 # Adapt these paths on Windows
 
-#Point this to the boost include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
-#Point this to the libtorrent include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
-#Point this to the zlib include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+# include
+INCLUDEPATH += $$quote(C:/qBittorrent/boost/boost)
```

Comment:
The first `boost` is where boost entire project relies. The second `boost` is where header files relies. I followed the wiki instructions so it would look like this...